cli/cmd/volume.go

```diff
@@ -248,16 +252,64 @@ var volumeResetCmd = &cobra.Command{
 	},
 }
 
+func volumeInfoHandler2(cmd *cobra.Command, isInfo bool) error {
+	var vols []api.Volinfo
+	var err error
+	validateNArgs(cmd, 0, 1)
+	volname := ""
+	if len(cmd.Flags().Args()) > 0 {
+		volname = cmd.Flags().Args()[0]
+	}
+	if volname == ""{
+		vols, err = client.Volumes("")
+	}else{
```

Comment:
Most of the code in cli packages for some reason aren't run through `gofmt`. Please do so. You can configure your editor to automatically run `gofmt`

Reply:
used gofmt. Please review the same. Thanks.